src/julia-parser.scm

```diff
@@ -1577,8 +1577,13 @@
                       (else #t)))
          (rest  (if done
                     '()
-                    (parse-comma-separated s (lambda (s)
-                                               (parse-import s word))))))
+                    (let ((ex (parse-comma-separated s (lambda (s)
+                                                               (parse-import s word)))))
+                      (if (eq? (peek-token s) ':)
+                          (error (string "unexpected \":\" in \"" word "\" syntax\n"
```

Comment:
How about `":" may only appear at the beginning of "<word>" syntax`?

Reply:
Maybe `":" in "<word>" syntax can only be used when importing a single module. Split imports into multiple lines.`? I think that way, it's a little bit clearer to new users, what exactly they should do to fix this.

Reply:
Ok.